docs/flox-5-minutes.md

```diff
@@ -5,637 +5,249 @@ description: Get started with creating Flox environments.
-7. Play around, see if everything works, what's missing, etc.
-8. Leave the environment via `exit` or Ctrl-D.
+## Get the project
 
-Let's walk through those steps and explain in more detail.
-
-## Install Flox
+We've prepared a sample project for you, but you'll need to [install Flox][install_flox] to follow along.
+Once you have Flox, you can clone the project:
 
-Start by [installing `flox`][install_flox].
-
-## Create an environment
-
-We'll start by creating an environment for your project,
-and we'll pretend that your project is called `my_project`.
-
-Flox installs packages into _environments_ as opposed to installing them into
-globally accessible directories in various places on your system.
-This allows you to have different versions of your tools across different
-projects without them interfering or conflicting.
-
-`cd` into your project and initialize the environment with [`flox init`][init].
-
-For Python, Node.js, and Go projects this will display a prompt asking if you
-would like Flox to automatically do some language-specific setup for you.
-
-=== "Python"
-
-    ```console
-    $ cd my_project
-
-    my_project $ flox init
-    Flox detected a Python project with the following Python provider(s):
-
-    * latest python (requirements.txt)
-
-      Installs latest python (3.12.5) with pip bundled.
-      Adds hooks to setup and use a venv.
-      Installs dependencies to the venv from: requirements.txt
-
-    ! Would you like Flox to set up a standard Python environment?
-    You can always change the environment's manifest with 'flox edit'
-    > Yes - with latest python
-      No
-      Show suggested modifications for latest python
-    [Use '--auto-setup' to apply Flox recommendations in the future.]
-    ...select yes...
-    ✨ Created environment 'my_project' (aarch64-darwin)
-    ✅ 'python3' installed to environment 'my_project'
-
-    Next:
-      $ flox search <package>    <- Search for a package
-      $ flox install <package>   <- Install a package into an environment
-      $ flox activate            <- Enter the environment
-      $ flox edit                <- Add environment variables and shell hooks
-    ```
-
-    If your Python project contains a `requirements.txt`,
-    a Poetry `pyproject.toml`, or a generic `pyproject.toml`,
-    you'll be asked whether you'd like Flox to install Python, `pip`, and
-    Poetry (if applicable).
-
-    If you accept, it will also create a virtual environment that's activated
-    automatically when you activate your Flox environment.
-
-    Note that the Flox Catalog already contains thousands of Python packages,
-    so you may not need `poetry`, `pipenv`, `pdm`, etc for package management.
-    That's right, with Flox you may be able to ditch your Python package
-    manager for something that works universally across all languages!
-
-=== "Node.js"
+<!-- markdownlint-disable MD014 -->
+```console
+$ git clone https://github.com/flox/flox-in-5min.git
+$ cd flox-in-5min
+```
```

Comment:
Given the change in https://github.com/flox/floxdocs/pull/311, let's format to be copy/paste friendly:

````suggestion
```{ .bash .copy }
git clone https://github.com/flox/flox-in-5min.git
cd flox-in-5min
```
````

Reply:
This won't copy properly without a semicolon at the end of the line, and I kind of hate how that looks. I think it's totally fine to have copyable one-liners, but it doesn't work well for multiline code blocks.

Reply:
Fair enough, let's table this for now.

Reply:
I'm wrong. I fixed this in a couple of places.